urdf_mesh_converter/package.xml

```diff
@@ -0,0 +1,30 @@
+<package format="3">
+
+  <name>urdf_mesh_converter</name>
+  <version>0.4.5</version>
+  <description>
+    Convert urdf mesh.
+  </description>
+
+  <license>BSD</license>
+  <maintainer email="yanokura@jsk.imi.i.u-tokyo.ac.jp">Iori Yanokura</maintainer>
+  <author email="yanokura@jsk.imi.i.u-tokyo.ac.jp">Iori Yanokura</author>
+
+  <buildtool_depend>catkin</buildtool_depend>
+  <build_depend>catkin_virtualenv</build_depend>
```

Comment:
Why is this not included in `ALPHABETICAL ORDER` lines?

Reply:
Thanks.
https://github.com/jsk-ros-pkg/jsk_model_tools/pull/255/commits/d3833088b7baea0223bba869ec7dc364d549c70d
Update package.xml for conditional Python 3 dependency

- Change build_depend and exec_depend to conditionally depend on python3 if ROS_PYTHON_VERSION is 3.
- Correct buildtool_depend from catkin to catkin_virtualenv.